spec/async_sender_spec.rb

```diff
@@ -54,7 +55,8 @@
 
       it "prints the debug message the correct number of times" do
         log = @stderr.string.split("\n")
-        expect(log.grep(/\*\*Airbrake: \{\}/).size).to eq(300)
+        expect(log.grep(/\*\*Airbrake: \{\}/).size).to \
+          be >= @sender.instance_variable_get(:@unsent).max
```

Comment:
Nitpick: could you please break it into two lines like this instead:

```ruby
expect(log.grep(/\*\*Airbrake: \{\}/).size).
  to be >= @sender.instance_variable_get(:@unsent).max
```

It is more consistent with the current style.